src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,18 @@
+import { ApiError } from '../exeptions/api.error.js';
+
+export const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
+
+  if (error) {
+    res.statusCode = 500;
+
+    res.send({
+      message: 'Server error',
+    });
+  }
```

Comment:
Consider using an `else` statement after the `if (error instanceof ApiError)` block to handle non-ApiError cases. This will make the logic clearer and ensure that only one error response is sent.